src/dataviews/histogram-dataview-model.js

```diff
@@ -71,6 +71,14 @@ module.exports = DataviewModelBase.extend({
     if (attrs && (attrs.min || attrs.max)) {
       this.filter && this.filter.setRange(this.get('min'), this.get('max'));
     }
+
+    if (this.get('customStart')) {
```

Comment:
It should be completely transparent, since those properties are added through the public api, and without them, the code should work as it did before, but yeah, we should be careful.